installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -1,18 +1,17 @@
-[id='proc-install-rhdh-airgapped-environment-ocp-helm_{context}']
-
-= Installing {product} using the Helm Chart in an air-gapped environment 
+[id="proc-install-rhdh-airgapped-environment-ocp-helm_{context}"]
+= Installing {product} using the Helm Chart in an air-gapped environment
 
 An air-gapped environment, also known as an air-gapped network or isolated network, ensures security by physically segregating the system or network. This isolation is established to prevent unauthorized access, data transfer, or communication between the air-gapped system and external sources.
 
 You can install {product} in an air-gapped environment to ensure security and meet specific regulatory requirements.
 
-To install {product-short} in an air-gapped environment, you must have access to the `registry.redhat.io` and the registry for the air-gapped environment. 
+To install {product-short} in an air-gapped environment, you must have access to the `registry.redhat.io` and the registry for the air-gapped environment.
 
 .Prerequisites
 
-* You have installed an {ocp-brand-name} ({ocp-short}) {ocp-version-min} or later.
+* You have installed an {ocp-brand-name} {ocp-version-min} or later.
 * You have access to the `registry.redhat.io`.
-* You have access to the {ocp-short} image registry of your cluster. For more information about exposing the {ocp-short} image registry, see https://docs.openshift.com/container-platform/{ocp-version}/registry/securing-exposing-registry.html[Exposing the registry] in OpenShift documentation.
+* You have access to the {ocp-brand-name} image registry of your cluster. For more information about exposing the image registry, see the {ocp-brand-name} documentation about https://docs.openshift.com/container-platform/{ocp-version}/registry/securing-exposing-registry.html[Exposing the registry].
```

Comment:
Per minimalism principle 1, we typically use the _For more information about [topic], see [link]_ format (so the _in OpenShift documentation_ was never even necessary) - just an fyi; definitely won't block the merge